ynput/OpenPype
# Launcher tool: Refactor launcher tool (for AYON)

## Changelog Description
Refactored launcher tool to new tool. Separated backend and frontend logic. Refactored logic is AYON-centric and is used only in AYON mode, so it does not affect OpenPype.

## Additional info
Added first models and widgets that can be commonly used at other places. Those are not yet used in workfiles tool as that deserves separated PR. Launcher tool should work the same way as did before except actions history button.

## Testing notes:
1. Create openpype addon
2. Upload addon to AYON server
3. Make sure you have right openpype addon downloaded in local files
4. Launch AYON launcher
5. Open launcher tool
6. Change context, launch actions
